### PR TITLE
Add aikidave to DOCS: Marketing approvers

### PR DESCRIPTION
**Pull request by aikithoughts**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Only IgorMinar and StephenFluin can approve DOCS: Marketing files
Issue Number: N/A

## What is the new behavior?
Extends approvers to include aikidave

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -779,6 +779,7 @@ groups:
       users:
         - IgorMinar
         - StephenFluin
+        - aikidave
```
Comment (IgorMinar) [inline .pullapprove.yml:782]: Can you please sort the list alphabetically? Thanks

**Review by IgorMinar — CHANGES_REQUESTED** [review]

**Review by IgorMinar — APPROVED** [review]
LGTM. Thanks!

Please keep in mind that with great powers also comes great responsibility! ❤️ 

Reviewed-for: code-ownership

**Comment by petebacondarwin** [comment]
@IgorMinar - don't forget to check the commit message is valid when reviewing... this should have been `docs:...` not `fix: ...`.

**Comment by aikidave** [comment]
Why `docs:...`? It's not really a docs fix?

**Comment by petebacondarwin** [comment]
Sorry I meant `build: ...`

**Comment by petebacondarwin** [comment]
or `ci: ...`

**Pull request closed without merging**